dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -3479,48 +2701,6 @@ public boolean isTwoRavenAmongExternalTools(List<ExternalTool> externalTools) {
         return cachedTools;
     }
 
-    Boolean thisLatestReleasedVersion = null;
-
-    public boolean isThisLatestReleasedVersion() {
-        if (thisLatestReleasedVersion != null) {
-            return thisLatestReleasedVersion;
-        }
-
-        if (!workingVersion.isPublished()) {
-            thisLatestReleasedVersion = false;
-            return false;
-        }
-
-        DatasetVersion latestPublishedVersion = null;
-        Command<DatasetVersion> cmd = new GetLatestPublishedDatasetVersionCommand(dvRequestService.getDataverseRequest(), dataset);
-        try {
-            latestPublishedVersion = commandEngine.submit(cmd);
-        } catch (Exception ex) {
-            // whatever...
-        }
-
-        thisLatestReleasedVersion = workingVersion.equals(latestPublishedVersion);
-
-        return thisLatestReleasedVersion;
-
-    }
-
-    public String getJsonLd() {
```

Comment:
I've got
```
        <ui:define name="jsonld_header">
            <script type="application/ld+json">
                <h:outputText value="#{DatasetPage.jsonLd}"/>
            </script>
        </ui:define>
```
in dataset.xhtml - looks like it is used

Reply:
Yes I noticed when I tried to create dataset, I re-added it in 2nd commit, it is just in different place.